nucliadb_node/src/telemetry.rs

```diff
@@ -41,35 +42,32 @@ pub fn init_telemetry() -> ServiceResult<()> {
             .install_batch(opentelemetry::runtime::Tokio)
             .map_err(|e| ServiceError::GenericErr(Box::new(e)))?;
 
-        // let filter = FilterFn::new(|metadata| {
-        //    let target = metadata.target();
-        //    match metadata.module_path() {
-        //        Some(module_path) if module_path.contains("nucliadb_node") => {
-        //            target.contains("nucliadb_node::writer")
-        //                || target.contains("nucliadb_node::reader")
-        //        }
-        //        _ => false,
-        //    }
-        //});
-        let filter = tracing_subscriber::EnvFilter::from_default_env();
+        let filter = FilterFn::new(|metadata| match metadata.file() {
```

Comment:
What do you think about writing the filter this way?
```rust
        let filter = FilterFn::new(|metadata| {
            metadata
                .file()
                .filter(|file| file.contains("nucliadb_node"))
                .map(|_| !metadata.is_event())
                .map(|state| state && !metadata.fields().field("trace_marker").is_none())
                .unwrap_or_default()
        });
```

Reply:
fixed